Improve rule file_permssions_crontab

jan-cerny opened the pull request:
This change solves problem of failing rule `file_permissions_crontab`
when the `crontabs` RPM package isn't installed. This situation happens
namely when builidng a CS9 bootable container image because the
base image doesn't contain the crontabs RPM package.

In profiles which contain rules checking permissions on files that are
provided by `crontabs`, we will install the `crontabs` package by
installing the `cronie` package which provides the `cron` service and
has `crontabs` as dependency. This will be achieved by adding the rule
`package_cron_installed` to these profiles.

Notice that the `cronie` package will be installed as a result of rule
`package_cron_installed` which installs either `cron` or `cronie`
based on the product name. This rule is fixed by this commit because
the correct package name for RHEL 9 is `cronie` as well.


#### Review Hints:

Build and boot a CS 9 bootable container image hardened with STIG profile. Then, run a verification scan of the running VM.

Mab879 commented on controls/stig_rhel9.yml:
```diff
@@ -958,6 +958,7 @@ controls:
             - medium
         title: RHEL 9 cron configuration directories must have a mode of 0700 or less permissive.
         rules:
+            - package_crontabs_installed
```
This isn't in the STIG. Why should be added?

See https://stigaview.com/products/rhel9/v2r2/RHEL-09-232040/

jan-cerny replied:
Good point! I have changed the PR to add `package_cron_installed` instead.

Mab879 reviewed (commented):
The changes look good other than the STIG changes.

jan-cerny commented:
I have removed the new rule from this PR.  I have changed the PR to add package_cron_installed instead.

I have rebased this PR on the top of the latest upstream master branch.

Mab879 commented on controls/stig_rhel9.yml:
```diff
@@ -958,6 +958,7 @@ controls:
             - medium
         title: RHEL 9 cron configuration directories must have a mode of 0700 or less permissive.
         rules:
+            - package_cron_installed
```
The STIG only calls out the permissions not the package install.

What problem are you trying to solve?

jan-cerny replied:
I'm trying to solve the problem that the rules checking these files fail because the files don't exist because the package that provides these files isn't installed.

Mab879 reviewed (commented):
I still have question on the STIG change.

Mab879 reviewed (changes requested):
The CI failures look valid.

Mab879 commented:
Waving Automatus tests since they are not applicable to containers.
